swap_meet/condition_dict.py

```diff
@@ -0,0 +1,14 @@
+''' 
+This module contains the condition table dictionary used by the Item class.
+The dictionary maps condition values to their corresponding descriptions. 
+It is used by the condition_description method in the Item class. 
+'''
+
+condition_table = {
+            5: "Brand New",
+            4: "Used - Like New/ Open Box/ Without Tags",
+            3: "Used - Good",
+            2: "Used - Fair",
+            1: "Used - Poor",
+            0: "For Parts - Not Working"
+        }
```

Comment:
How love you used a dictionary here! This could also be a time to use match case! 
```python
    def condition_description(self):
        match self.condition:
            case 0:
                return "Throw Away"
            case 1:
                return "Eh"
            case 2:
                return "Better..."
            case 3:
                return "..."
            case 4: 
                return "Okay?!"
            case 5:
                return "Yes!"
```